Backpack/Calendar/Classes/BPKCalendarSelectionConfigurationRange.h

```diff
@@ -22,6 +22,18 @@ NS_ASSUME_NONNULL_BEGIN
 
 @interface BPKCalendarSelectionConfigurationRange : BPKCalendarSelectionConfiguration
```

Comment:
None of the selection configurations are documented which means they'll not show up on the docs site